spec/hbw/features/bp_form/translation_spec.rb

```diff
@@ -1,5 +1,6 @@
 feature 'BP form with existing russian translation', js: true do
   before(:each) do
+    set_mock_file('spec/hbw/features/bp_form/translation_mock.yml')
```

Comment:
I think it is not too hard to use fool path to mock file. So this approach to set mock file is absolutely explicit